tests/integration/ha/continuous_writes.py

```diff
@@ -133,12 +134,19 @@ async def max_stored_id(self) -> int:
     async def _create_fully_replicated_index(self):
         """Create index with 1 p_shard and an r_shard on each node."""
         client = await self._client()
+        units = await get_application_unit_ips(self._ops_test, app=self._app)
+
         try:
             # create index with a replica shard on every node
             client.indices.create(
                 index=ContinuousWrites.INDEX_NAME,
                 body={
-                    "settings": {"index": {"number_of_shards": 2, "auto_expand_replicas": "1-all"}}
```

Comment:
let's not change this please. Multiple tests rely on the health of the cluster as per this configuration.

Reply:
B/c GH runners are small, I need to run microceph + OpenSearch in 2x units only. Therefore, I need here to be flexible.

Reply:
@Mehdi-Bendriss I will roll this change back as we have more powerful GH runners now.